packages/caliper-tests-integration/fabric_tests/run.sh

```diff
@@ -37,7 +37,7 @@ fi
 export CALIPER_PROJECTCONFIG=../caliper.yaml
 
 dispose () {
-    ${CALL_METHOD} launch master --caliper-workspace phase5 --caliper-flow-only-end
+    ${CALL_METHOD} launch master --caliper-workspace phase7 --caliper-flow-only-end
```

Comment:
due to using reduce network config info, this requires the `--caliper-fabric-gateway-enabled` flag too

Reply:
Not sure I follow. Only the end phase is executed, and the validator takes that into account. The logs look fine: https://travis-ci.com/github/hyperledger/caliper/jobs/353155886#L1512
Or am I missing something? 🤔

Reply:
Jut tested locally, and I was mistaken...